include/drivers/ipm.h

```diff
@@ -87,12 +143,104 @@ typedef void (*ipm_register_callback_t)(const struct device *port,
  */
 typedef int (*ipm_set_enabled_t)(const struct device *ipmdev, int enable);
 
+/**
+ * @typedef ipm_max_channels_get_t
+ * @brief Callback API to get maximum number of channels
+ *
+ * See @a ipm_max_channels_get() for argument definitions.
+ */
+typedef uint32_t (*ipm_max_channels_get_t)(const struct device *ipmdev);
+
+/**
+ * @typedef ipm_configure_channel_t
+ * @brief Configure (enable/disable) channel in the IPM device
```

Comment:
`ipm_configure_channel()` does not mention dsiabling channels in its description. Perhaps this could be aligned.

Reply:
Fixed.